server/ladder_service.py

```diff
@@ -289,34 +287,35 @@ def inform_player(self, player: Player):
                     )
                 })
 
-    def start_queue_handlers(self):
-        for queue in self.queues.values():
-            asyncio.ensure_future(self.handle_queue_matches(queue))
+    def on_match_found(
+        self,
+        s1: Search,
+        s2: Search,
+        queue: MatchmakerQueue
+    ) -> None:
+        try:
+            msg = {"command": "match_found", "queue": queue.name}
 
-    async def handle_queue_matches(self, queue: MatchmakerQueue):
-        async for s1, s2 in queue.iter_matches():
-            try:
-                msg = {"command": "match_found", "queue": queue.name}
-                # TODO: Handle disconnection with a client supported message
-                await asyncio.gather(*[
-                    player.send_message(msg)
-                    for player in s1.players + s2.players
-                ])
-                asyncio.create_task(
-                    self.start_game(s1.players, s2.players, queue)
-                )
-            except Exception as e:
-                self._logger.exception(
-                    "Error processing match between searches %s, and %s: %s",
-                    s1, s2, e
-                )
+            for player in s1.players + s2.players:
```

Comment:
operator `+`, is there magic inside? I'm offering you to not to override magic methods like `def __add__`.

Reply:
No. it's list + list.